OnlineCinema.API/Controllers/GenreMVCController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using OnlineCinema.BL.Extensions;
+using OnlineCinema.BL.Model;
+using OnlineCinema.BL.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace OnlineCinema.API.Controllers
+{
+    public class GenreMVCController : Controller
+    {
+        private readonly IGenreService _genreService = new GenreService();
+
+        public ActionResult Index()
+        {
+            var genres = _genreService.GetAll().OrderBy(f => f.Name);
+            return View(genres);
+        }
+
+        [HttpGet]
+        public ActionResult Editor(int id)
+        {
+            if(id == 0)
+            {
+                return View("Edit", new GenreView());
```

Comment:
No need in "Edit" parameter. Rename method to Edit.

If you have edit action already, just play with HttpMethod.